README.md

```diff
@@ -82,49 +76,43 @@ conn.postTransaction(txSigned)
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title>HTML5 boilerplate – all you really need…</title>
+        <title>BigchainDB boilerplate</title>
         <!-- Adjust version to your needs -->
         <script src="https://unpkg.com/bigchaindb-driver@0.1.1/dist/browser/bigchaindb-driver.window.min.js"></script>
-        <script>
 
+        <script>
             // BigchainDB server instance or IPDB (e.g. https://test.ipdb.io/api/v1/)
             const API_PATH = 'http://localhost:9984/api/v1/'
 
-            // Create a new user with a public-private key pair
-            // (or a whole bunch of them, nobody's counting)
-            const alice = new BigchainDB.Ed25519Keypair()
+            // Create a new keypair.
+            const alice = new driver.Ed25519Keypair()
```

Comment:
only `window.BigchainDB` is available here.

I suggest to use `new BigchainDB.<methodname>()`